client/packages/purchasing/src/purchase_order/DetailView/Toolbar.tsx

```diff
@@ -39,6 +43,7 @@ export const Toolbar = ({ isDisabled }: ToolbarProps) => {
     handleChange,
   } = usePurchaseOrder();
   const { updateLines } = usePurchaseOrderLine();
+  const { convertByRate } = useCurrencyConversion();
```

Comment:
We don't need this right? The price that is being shown in purchase orders is already being displayed in the foreign currency.. OG doesn't seem to convert prices even if you change the rate 👀  please confirm @craigdrown @mark-prins

Reply:
I did noticed that in OG as well. Can remove my changes if needed. This was just my interpretation of this requirement (but yea might be wrong):

<img width="616" height="57" alt="Screenshot 2025-09-29 at 12 14 33 PM" src="https://github.com/user-attachments/assets/aac40fb1-4002-4864-9021-3392cc2f5838" />

Reply:
Thanks Craig. Will remove conversion logic shortly

Reply:
Please see fix here: https://github.com/msupply-foundation/open-msupply/pull/9343/commits/9e68fcbe89c7d77eb8477015d5ec3b8af8911f43